src/visualization/components/Snapshot.tsx

```diff
@@ -28,9 +32,14 @@ class Snapshot extends React.Component<any, any> {
       var events = this.props.impression.events;
       var start = events[0].time;
       var end = events[events.length - 1].time;
+      
+      
```

Comment:
whitespace

Reply:
not resolved - if you disagree mark as won'tfix and give an explanation

---
In reply to: [175851523](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175851523) [](ancestors = 175851523)